MAUI/Maps/vector-layers/polyline-layer.md

```diff
@@ -426,4 +426,90 @@ this.Content = maps;
 
 {% endtabs %}
 
-![Polyline dash array](images/polyline-layer/polyline-dash-array.png)
\ No newline at end of file
+![Polyline dash array](images/polyline-layer/polyline-dash-array.png)
+
+## Animation
+
+You can apply animation for the [`MapPolyline`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapPolyline.html) using the `MapPolylineLayer.AnimationDuration` and `MapPolylineLayer.AnimationEasing` properties and able to customize the animation curve and duration.
```

Comment:
You can apply animation for the [`MapPolyline`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapPolyline.html) using the `MapPolylineLayer.AnimationDuration` and `MapPolylineLayer.AnimationEasing` properties and be able to customize the animation curve and duration.

Reply:
Updated the content